vinod-jacob/rdkservices
# XIONE-1313: Add DisplayInfo implementation for Realtek

Reason for change:
* Add DisplayInfo implementation for Realtek SOC
* Implement SoC_GetGraphicsWidth()/SoC_GetGraphicsHeight() using libdrm
* Implement SoC_GetTotalGpuRam()/SoC_GetFreeGpuRam() using MemTotal: and MemFree: field
  from /proc/meminfo because Realtek platform Mali GPU has MMU and use the
  system memory

Test Procedure:
* Add DisplayInfo in PACKAGECONFIG in rdkservices_git.bbappend for Realtek SOC layer
  and build RDKServices, DisplayInfo plug-in should be build and works.
Risks: Low

Change-Id: I48f013e2eb3819af41a83e2d4c6bd6049f1bb628
Signed-off-by: Mark Yang <mark.yang@realtek.com>
